### ISSUE TITLE
Save files with the original name

### ISSUE THREAD

**Issue by nunoarruda** [nunoarruda/auto-save-html#11]
A user requested the following to me via email:
```
Hi Nuno !

Thank you so much for your extension !
Do you think it's possible to give the real name of the file (like www.mywebsite.com/index.html -> index.html) instead of '' AutoSave_{TIMESTAMP}.htm'' ?

Thank you,

Regards,
```

This is technically possible but it would probably do more harm than good (files with the same name, no order, etc). I have no intention of implementing this but decided to open an issue anyways in case there's demand from other users.

**Comment by lagrandeours** [comment]
may be it s possible de add the original file name at the end ... ?

**Comment by nunoarruda** [comment]
Yeah, it's possible. This is just a case of evaluating the pros and cons and deciding if it's worth implementing such a thing.
